src/segments/python.go

```diff
@@ -63,21 +63,26 @@ func (p *Python) loadContext() {
 	if !p.language.props.GetBool(FetchVirtualEnv, true) {
 		return
 	}
-	venvVars := []string{
-		"VIRTUAL_ENV",
-		"CONDA_ENV_PATH",
-		"CONDA_DEFAULT_ENV",
-	}
-	var venv string
-	for _, venvVar := range venvVars {
-		venv = p.language.env.Getenv(venvVar)
-		if len(venv) == 0 {
-			continue
+	prompt := p.pyvenvCfgPrompt()
```

Comment:
This can be an early return instead a full if/else block:

```go
prompt := p.pyvenvCfgPrompt()
if len(prompt) > 0 {
    p.Venv = prompt
    return
}
```